<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>571</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 182);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>411</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-10</y>
        <width>391</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bahnschrift</family>
        <pointsize>21</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 255, 255, 255), stop:0.495 rgba(255, 255, 255, 255), stop:0.505 rgba(255, 0, 0, 255), stop:1 rgba(255, 0, 0, 255));

color: rgb(127, 57, 251);</string>
      </property>
      <property name="text">
       <string>Resistencias en serie y paralelo</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>360</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="calular3">
         <property name="styleSheet">
          <string notr="true">QPushButton{
font: 63 16pt &quot;Bahnschrift SemiBold SemiConden&quot;;
font: 16pt &quot;Bahnschrift SemiLight Condensed&quot;;
font: 25 16pt &quot;Bahnschrift Light SemiCondensed&quot;;
font: 25 16pt &quot;Bahnschrift Light SemiCondensed&quot;;
font: 12pt &quot;Bahnschrift SemiLight&quot;;
color: white;
border-radius: 5px;
background-color: rgb(187, 134, 252);
}

QPushButton:hover{
background-color: rgb(199, 199, 199);
color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string>Calcular</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="respuesta3">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid  rgb(143, 142, 142);
color: rgb(107, 105, 105);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="volverrsyp">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
font: 63 16pt &quot;Bahnschrift SemiBold SemiConden&quot;;
font: 16pt &quot;Bahnschrift SemiLight Condensed&quot;;
font: 25 16pt &quot;Bahnschrift Light SemiCondensed&quot;;
font: 25 16pt &quot;Bahnschrift Light SemiCondensed&quot;;
font: 12pt &quot;Bahnschrift SemiLight&quot;;
color: white;
border-radius: 5px;
background-color: rgb(187, 134, 252);
}

QPushButton:hover{
background-color: rgb(199, 199, 199);
color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>381</width>
        <height>261</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>imagenes/Sin título.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="r1pys">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>49</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid  rgb(143, 142, 142);
color: rgb(107, 105, 105);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>R1</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="r2pys">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>49</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid  rgb(143, 142, 142);
color: rgb(107, 105, 105);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>R2</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="r3pys">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>120</y>
        <width>49</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid  rgb(143, 142, 142);
color: rgb(107, 105, 105);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>R3</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="r4pys">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>49</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid  rgb(143, 142, 142);
color: rgb(107, 105, 105);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>R4</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="r5pys">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>260</y>
        <width>49</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
border: 1px solid  rgb(143, 142, 142);
color: rgb(107, 105, 105);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>R5</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>260</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(kΩ)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>260</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(kΩ)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>170</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(kΩ)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(kΩ)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>(kΩ)</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
